src/backend/egl/ffi.rs

```diff
@@ -130,6 +130,7 @@ pub mod egl {
     include!(concat!(env!("OUT_DIR"), "/egl_bindings.rs"));
 
     pub const RESOURCE_BUSY_EXT: u32 = 0x3353;
+    pub const PLATFORM_XCB_EXT: u32 = 0x31DC;
```

Comment:
Manually adding these is annoying, can't just update upstream since the egl.xml isn't parsed correctly.

Might want to look into phosphorus at some point and finish that egl pull request for that?

Reply:
Well, at least its not much..